test/expected/create_hypertable.out

```diff
@@ -220,10 +240,10 @@ SELECT * FROM _timescaledb_internal.get_create_command('test_1dim');
 (1 row)
 
 \dt "test_schema".*
-                         List of relations
-   Schema    |          Name          | Type  |        Owner        
--------------+------------------------+-------+---------------------
- test_schema | test_1dim              | table | default_perm_user_2
```

Comment:
Why this change? Do we want to switch back to `default_perm_user_2` on line 227(new line #s)

Reply:
Above I test the case of having one user create chunks in another user's (associated) schema without appropriate permission. This required switching to another role. Figured it didn't matter which of the non-superusers that the rest of the tests run as. Switching back would be an extra line adding nothing to the tests just to make this line the same as before.

If you think it matters or is important I can add a `SET ROLE` above.

Reply:
OK